aerospike/aerospike-client-python
# [CLIENT-3186] CI/CD: Use entrypoint script to finish setting up strong consistency instead of the run-ee-server composite action

Testing

- We can manually deploy the server with `docker build` and `docker run`, then run the MRT tests successfully

Extra changes:

- Add healthcheck to check that container is ready to use through Github Actions or locally on the command line
- For our custom EE image, we check the container health status before running tests against the server
- wait-for-as-server-to-start composite action has been repurposed to wait for the base CE server to start (renamed to "wait-for-ce-server-to-start")
- wait-for-ce-server-to-start composite action now fails if we cannot confirm that the server is ready to use for testing within 30 seconds
